utils/scapp.h

```diff
@@ -545,6 +543,11 @@ class scApp : public ktxApp {
           "                   medium     | (equivalent to quality =  60)\n"
           "                   thorough   | (equivalent to quality =  98)\n"
           "                   exhaustive | (equivalent to quality = 100)\n"
+          "      --astc_perceptual\n"
+          "               The codec should optimize perceptual error, instead of direct\n"
```

Comment:
s/optimize perceptual/optimize for perceptual/

Reply:
done in https://github.com/KhronosGroup/KTX-Software/pull/511/commits/f7dc5c23320d9096b782aae1812317a8c62fad07

Reply:
Thanks.